LFPy/eegmegcalc.py

```diff
@@ -470,11 +518,18 @@ def _potential_csf_tan(self, r, theta):
             Summation factor for calculation of electrical potential in CSF
             from tangential current dipole moment. (unitless)
         """
-        n = np.arange(1,100)
-        c2n = self._calc_c2n(n)
-        d2n = self._calc_d2n(n, c2n)
-        consts = c2n*(r/self.r2)**n + d2n*(self.r2/r)**(n+1)
-        pot_sum = np.sum([c*lpmv(1, i, np.cos(theta)) for c,i in zip(consts,n)])
+        n = 1
+        const = 1.
+        coeff_sum = 0.
+        consts = []
+        while const > 2./99.*1e-6*coeff_sum:
```

Comment:
It's getting repetitive. Actually, I suppose this new parameter for this convergence criteria must be made a class input.